src/compat/db_vdb.c

```diff
@@ -2308,6 +2364,87 @@ char_array_to_name_list (PARSER_CONTEXT * parser, char **names, int length)
   return list;
 }
 
+static DB_PREPARE_CTE_INFO *
+set_prepare_cte_info (PT_NODE * cte_query, DB_PREPARE_CTE_INFO * cte_info, int cte_num_query)
+{
+  int i, q = cte_num_query;
+  int alloc_size = sizeof (DB_PREPARE_CTE_INFO) * (cte_num_query / 4 + 1) * 4;
```

Comment:
alloc_size is used only the in the following if statement block

Reply:
Ok. good point.